manim/renderer/opengl_renderer.py

```diff
@@ -43,15 +42,26 @@ def __init__(
         focal_distance=2,
         light_source_position=[-10, 10, 10],
         orthographic=False,
+        minimum_polar_angle=-PI / 2,
+        maximum_polar_angle=PI / 2,
```

Comment:
I'm not sure why, but these limits don't do anything when using 3b1b's shaders, only the projection ones.

Reply:
Pretty much none of the OpenGL features I wrote will work with those shaders. They don't follow any of the same standards that pretty much every other shader library uses.